charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -122,7 +122,6 @@ spec:
             - --csi-address=$(ADDRESS)
             - --v=5
             - --leader-election=true
-            - --leader-election-type=leases
```

Comment:
we don't need this because the default changed to leases right? just checking

Reply:
or they removed the option an dit's leases only?

Reply:
Yeah they removed it and it's leases.